website/README.md

```diff
@@ -554,6 +554,40 @@ extensions for your code editor. The
 [`.prettierrc`](https://github.com/explosion/spaCy/tree/master/website/.prettierrc)
 file in the root defines the settings used in this codebase.
 
+## Building & Developing the Site with Docker {#docker}
+Sometimes it's hard to get a local environment working due to rapid updates to node dependencies,
+so it may be easier to use docker for building the docs.
+
+If you'd like to do this,
+**be sure you do *not* include your local `node_modules` folder**,
+since there are some dependencies that need to be built for the image system. 
+Rename it before using.
+
+```bash
+docker run -it \
+  -v $(pwd):/spacy-io/website \
+  -p 8000:8000 \
+  ghcr.io/explosion/spacy-io \
+  gatsby develop -H 0.0.0.0
+```
+
+This will allow you to access the built website at http://0.0.0.0:8000/ 
+in your browser, and still edit code in your editor while having the site 
+reflect those changes.
+
+**Note**: On M1 Macs you may need to the image tagged `arm64` (`ghcr.io/explosion/spacy-io:arm64`),
```

Comment:
there's a verb or something missing here?

Reply:
It makes sense to me but it came from my brain so that's not a great test :) I always favor more clarity. How about:

> If you're working on a Mac with an M1 processor, you might see segfault errors from `qemu` if you use the default image. To fix this, use the `arm64` tagged image in the `docker run` command (`ghcr.io/explosion/spacy-io:arm64`).

Reply:
so it was missing the verb "use"? (I'm trying really hard to parse your original sentence but I really can't, haha, it may just be me)

Reply:
I'll reformulate to my new suggestion!

To me it's implied that they would "use" it in the original sentence, like if I gave someone an umbrella before they went outside I'd say "You might need this".

Reply:
But but but it said "may need to the image", not "may need the image" 🙈 
(it's not relevant, at all, I'm just so confused)

Reply:
Well then this is definitely a case of me failing to read my own writing correctly 🙈